caput/pipeline/tasklib/base.py

```diff
@@ -399,7 +436,7 @@ def next(self, *input):
         return output if len(output) > 1 else output[0]
 
     def finish(self):
-        """Should not need to override. Implement `process_finish` instead."""
+        """Should not need to override. Implement :py:meth:`~.ContainerTask.process_finish` instead."""
```

Comment:
The class docstring says this can be optionally overriden.

Reply:
I think the class docstring is wrong, so I've changed that